keyboards/handwired/split65/stm32/halconf.h

```diff
@@ -0,0 +1,531 @@
+/*
```

Comment:
please run `qmk chibios-confmigrate` on these files and post the small form of these files